Add sanity to track pcc drop due to inplace updation in swinv2 model

kamalrajkannan78 opened the pull request:
### summary 

- Sanity added to track pcc drop due to inplace updation in swinv2 model. for complete context pls take look a look on https://github.com/tenstorrent/tt-forge-fe/issues/1793

vkovinicTT commented:
As far as I know we should avoid using inplace operations as they can have unpredictable behaviour when doing backward pass. Is there a way to update the model so that we change that inplace operation with out of place equivalent?

kamalrajkannan78 commented:
> As far as I know we should avoid using inplace operations as they can have unpredictable behaviour when doing backward pass. Is there a way to update the model so that we change that inplace operation with out of place equivalent?

@vkovinicTT  please have a look on description of this [issue](https://github.com/tenstorrent/tt-forge-fe/issues/1793)

vkovinicTT commented:
@pilkicTT should we add this sanity test to `test_eltwise_unary.py`?

kamalrajkannan78 commented:
> > @pilkicTT is it okay to leave this test without `push` mark since this is just sanity check? Other than that it looks fine.
> 
> We can run it, it is a small test...

@vkovinicTT @pilkicTT 

To avoid changes in pytest.ini , I just added push marker.

pilkicTT commented:
> > > @pilkicTT is it okay to leave this test without `push` mark since this is just sanity check? Other than that it looks fine.
> > 
> > 
> > We can run it, it is a small test...
> 
> @vkovinicTT @pilkicTT
> 
> To avoid changes in pytest.ini , I just added push marker.

I am pretty sure that's not enough... just check the test logs, to confirm if your test is being executed

kamalrajkannan78 commented:
> > > > @pilkicTT is it okay to leave this test without `push` mark since this is just sanity check? Other than that it looks fine.
> > > 
> > > 
> > > We can run it, it is a small test...
> > 
> > 
> > @vkovinicTT @pilkicTT
> > To avoid changes in pytest.ini , I just added push marker.
> 
> I am pretty sure that's not enough... just check the test logs, to confirm if your test is being executed

Sry I missed the context that I added a new file. Now I added path in pytest.ini too

kamalrajkannan78 commented:
> > @vkovinicTT @pilkicTT
> > To avoid changes in pytest.ini , I just added push marker.
> 
> I am pretty sure that's not enough... just check the test logs, to confirm if your test is being executed

After adding the path of the new script in pytest.ini, it ran on push CI.
